microsoft-365/enterprise/PortalLaunchScheduler.md

```diff
@@ -21,81 +21,116 @@ description: "This article describes how you can launch your portal using the Po
 
 For more information about launching a successful portal, follow the basic principles, practices, and recommendations detailed in the [Creating, launching and maintaining a healthy portal](https://go.microsoft.com/fwlink/?linkid=2105838). 
 
-## App setup
-1. Uninstall if there an existing `Microsoft.Online.SharePoint.PowerShell` from your machine through the control panel.
-2. On PowerShell pass `Install-Module -Name Microsoft.Online.SharePoint.PowerShell`.
+> [!NOTE]
+> This feature is not available for Office 365 Germany, Office 365 operated by 21Vianet (China), or Microsoft 365 US Government plans.
+
+## App setup and connecting to SharePoint Online
+1. [Download the latest SharePoint Online Management Shell](https://go.microsoft.com/fwlink/p/?LinkId=255251).
+
+    > [!NOTE]
+    > If you installed a previous version of the SharePoint Online Management Shell, go to Add or remove programs and uninstall "SharePoint Online Management Shell." <br>On the Download Center page, select your language and then click the Download button. You'll be asked to choose between downloading a x64 and x86 .msi file. Download the x64 file if you're running the 64-bit version of Windows or the x86 file if you're running the 32-bit version. If you don't know, see [Which version of Windows operating system am I running?](https://support.microsoft.com/help/13443/windows-which-operating-system). After the file downloads, run it and follow the steps in the Setup Wizard.
+
+2. Connect to SharePoint as a [global admin or SharePoint admin](/sharepoint/sharepoint-admin-role) in Microsoft 365. To learn how, see [Getting started with SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online).
+
+
+## View any existing portal launch setups
+
+To see if there are existing portal launch configurations:
+
+   ```PowerShell
+   Get-SPOPortalLaunchWaves -LaunchSiteUrl <object> -DisplayFormat <object>
+   ```
+
+## Schedule a portal launch on the site
+
+The number of waves required depends on your expected launch size. 
+- Less than 10k users: 1 wave
+- 10k to 30k users: 3 waves 
+- 30k+ to 100k users: 5 waves
+- More than 100k users: 5 waves and contact your Microsoft account team
+
+### Steps for bi-directional redirection
+
+Bidirectional redirection involves launching a new modern SharePoint Online portal to replace an existing SharePoint classic or modern portal. Users in active waves will be redirected to the new site regardless of whether they navigate to the old or new site. Users in a non-launched wave that try to access the new site will be redirected back to the old site until their wave is launched. Should you have administrators or owners that need access to the old and new sites without being redirected, ensure they are listed using the `WaveOverrideUsers` parameter. 
+
```

Comment:
Need to add a sentence: "We only support redirection between the default home page on the old site and the default home page on the new site."